src/modsim2/data/loader.py

```diff
@@ -365,29 +361,40 @@ def _split_indices(
             cifar=self.cifar,
         )
 
-    def compute_similarity(self, only_train: bool = False):
+    def compute_similarity(
+        self, only_train: bool = False, metric_seed: int = 42
```

Comment:
Set default argument for `metric_seed` to `self.seed`

Reply:
@philswatton Should `metric_seed` be an argument in this method at all? e.g. do we ever want to set it independently of the dmpair seed? 

Also a note: it isn't possible to set the default directly to `self.seed` (get error name 'self' not defined), so would need to set `metric_seed: int = None` in the method definition and then assign `metric_seed` to `self.seed` if `metric_seed is None`.

Reply:
Think I have answered my question on this one - as it is possible to call compute_similarity multiple times for one DMPair, then it makes sense to be able to set the `metric_seed` independently of the object's seed.